content/en/serverless/installation/nodejs.md

```diff
@@ -198,15 +198,15 @@ For example:
 datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -v 26 --forwarder arn:aws:lambda:us-east-1:000000000000:function:datadog-forwarder
 ```
 
-If your Lambda function is configured to use code signing, you must add Datadog's Signing Profile ARN (`arn:aws:signer:us-east-1:464622532012:/signing-profiles/DatadogLambdaSigningProfile/9vMI9ZAGLc`) to your function's [Code Signing Configuration][5] before you can instrument it with the Datadog CLI. 
+If your Lambda function is configured to use code signing, you must add Datadog's Signing Profile ARN (`arn:aws:signer:us-east-1:464622532012:/signing-profiles/DatadogLambdaSigningProfile/9vMI9ZAGLc`) to your function's [Code Signing Configuration][4] before you can instrument it with the Datadog CLI. 
```

Comment:
Side note - can we remove this from the main installation steps into a separate file? I think since this doesn't apply to most customers this will come off as confusing ("This seems complicated, I don't know if I should set this up") and makes our happy path installation instructions longer.

Reply:
We should remove this for all runtimes, a potential solution is to add a new Code Signing page under the "Serverless Integrations" section.

Reply:
Perhaps we can shorten the paragraph and link it to a separate page, but I don't think we should remove this paragraph completely here, because for customers having code signing enforcement enabled this is a must-do step. Of course, it's a relatively new feature so it's less important right now, but I don't know if it's going to be adopted more widely in the future. Either way, I think this should be a separate PR, and probably we should also consult @nhinsch's opinion?

Reply:
My understanding from takes from serverless devs online is this something more niche that people in enterprises know to look for to check an audit/compliance checkbox (as opposed to something they aren't sure whether or not they need), so I think there's more benefit to putting this in a separate page so our instructions are more straightforward.